pages/common/find.md

```diff
@@ -28,7 +28,7 @@
 
 - Run a command for each file (use `{}` within the command to access the filename):
 
-`find {{root_path}} -name '{{*.ext}}' -exec {{wc -l {} }}\;`
+`find {{root_path}} -name '{{*.ext}}' -exec {{wc -l {} }} \;`
```

Comment:
Oh oops! I've just noticced that we now have a double space here - one inside the closing `}}` of the tldr-pages `{{token_syntax}}`, and the other now outside. I think this was fine as it was:

![Selection_2020-11-27_01:52:10_001_2dc5b7d](https://user-images.githubusercontent.com/9929737/100402033-2c627180-3053-11eb-811e-d95d311cfaaf.png)